--- source/src/cmd.c
+++ newdir/src/cmd.c
@@ -876,12 +876,14 @@
 STATIC_PTR int
 wiz_wish(VOID_ARGS) /* Unlimited wishes for debug mode by Paul Polderman */
 {
-    if (wizard) {
+    if (TRUE || wizard) {
         boolean save_verbose = flags.verbose;
-
+        boolean save_wizard = wizard;
         flags.verbose = FALSE;
+	wizard = TRUE;
         makewish();
         flags.verbose = save_verbose;
+	wizard = save_wizard;
         (void) encumber_msg();
     } else
         pline(unavailcmd, visctrl((int) cmd_from_func(wiz_wish)));
@@ -892,7 +894,9 @@
 STATIC_PTR int
 wiz_identify(VOID_ARGS)
 {
-    if (wizard) {
+    if (TRUE || wizard) {
+        boolean save_wizard = wizard;
+        wizard = TRUE; 
         iflags.override_ID = (int) cmd_from_func(wiz_identify);
         /* command remapping might leave #wizidentify as the only way
            to invoke us, in which case cmd_from_func() will yield NUL;
@@ -903,6 +907,7 @@
             iflags.override_ID = C('I');
         (void) display_inventory((char *) 0, FALSE);
         iflags.override_ID = 0;
+        wizard = save_wizard;
     } else
         pline(unavailcmd, visctrl((int) cmd_from_func(wiz_identify)));
     return 0;
@@ -1000,9 +1005,11 @@
 STATIC_PTR int
 wiz_map(VOID_ARGS)
 {
-    if (wizard) {
+    if (TRUE || wizard) {
         struct trap *t;
         long save_Hconf = HConfusion, save_Hhallu = HHallucination;
+        boolean save_wizard = wizard;
+        wizard = TRUE;
 
         HConfusion = HHallucination = 0L;
         for (t = ftrap; t != 0; t = t->ntrap) {
@@ -1012,6 +1019,7 @@
         do_mapping();
         HConfusion = save_Hconf;
         HHallucination = save_Hhallu;
+        wizard = save_wizard;
     } else
         pline(unavailcmd, visctrl((int) cmd_from_func(wiz_map)));
     return 0;
@@ -1021,9 +1029,12 @@
 STATIC_PTR int
 wiz_genesis(VOID_ARGS)
 {
-    if (wizard)
+    if (TRUE || wizard) {
+        boolean save_wizard = wizard;
+        wizard = TRUE;
         (void) create_particular();
-    else
+        wizard = save_wizard;
+    } else
         pline(unavailcmd, visctrl((int) cmd_from_func(wiz_genesis)));
     return 0;
 }
@@ -1043,9 +1054,12 @@
 STATIC_PTR int
 wiz_detect(VOID_ARGS)
 {
-    if (wizard)
+    if (TRUE || wizard) {
+        boolean save_wizard = wizard;
+        wizard = TRUE;
         (void) findit();
-    else
+        wizard = save_wizard;
+    } else
         pline(unavailcmd, visctrl((int) cmd_from_func(wiz_detect)));
     return 0;
 }
@@ -1054,9 +1068,12 @@
 STATIC_PTR int
 wiz_level_tele(VOID_ARGS)
 {
-    if (wizard)
+    if (TRUE || wizard) {
+        boolean save_wizard = wizard;
+        wizard = TRUE;
         level_tele();
-    else
+        wizard = save_wizard;
+    } else
         pline(unavailcmd, visctrl((int) cmd_from_func(wiz_level_tele)));
     return 0;
 }
@@ -1144,7 +1161,10 @@
 STATIC_PTR int
 wiz_polyself(VOID_ARGS)
 {
+    boolean save_wizard = wizard;
+    wizard = TRUE;
     polyself(1);
+    wizard = save_wizard;
     return 0;
 }
 
@@ -4323,7 +4343,7 @@
     int mode = BASICENLIGHTENMENT;
 
     /* show more--as if final disclosure--for wizard and explore modes */
-    if (wizard || discover)
+    if (TRUE || wizard || discover)
         mode |= MAGICENLIGHTENMENT;
 
     enlightenment(mode, ENL_GAMEINPROGRESS);
@@ -4752,7 +4772,7 @@
 #else
     { '\0', "levelchange", "ŒoŒ±ƒŒƒxƒ‹‚ð•Ï‚¦‚é",
 #endif
-            wiz_level_change, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_level_change, IFBURIED | AUTOCOMPLETE },
 #if 0 /*JP:T*/
     { '\0', "lightsources", "show mobile light sources",
 #else
@@ -4813,7 +4833,7 @@
 #else
     { '\0', "polyself", "•Ï‰»‚·‚é",
 #endif
-            wiz_polyself, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_polyself, IFBURIED | AUTOCOMPLETE },
 #if 0 /*JP:T*/
     { M('p'), "pray", "pray to the gods for help",
 #else
@@ -4986,19 +5006,19 @@
             wiz_debug_cmd_bury, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
 #endif
     { C('e'), "wizdetect", "reveal hidden things within a small radius",
-            wiz_detect, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_detect, IFBURIED | AUTOCOMPLETE },
     { C('g'), "wizgenesis", "create a monster",
-            wiz_genesis, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_genesis, IFBURIED | AUTOCOMPLETE },
     { C('i'), "wizidentify", "identify all items in inventory",
-            wiz_identify, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_identify, IFBURIED | AUTOCOMPLETE },
     { '\0', "wizintrinsic", "set an intrinsic",
             wiz_intrinsic, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
     { C('v'), "wizlevelport", "teleport to another level",
-            wiz_level_tele, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_level_tele, IFBURIED | AUTOCOMPLETE },
     { '\0', "wizmakemap", "recreate the current level",
             wiz_makemap, IFBURIED | WIZMODECMD },
     { C('f'), "wizmap", "map the level",
-            wiz_map, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_map, IFBURIED | AUTOCOMPLETE },
 #if 0 /*JP:T*/
     { '\0', "wizrumorcheck", "verify rumor boundaries",
 #else
@@ -5014,7 +5034,7 @@
     { '\0', "wizwhere", "show locations of special levels",
             wiz_where, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
     { C('w'), "wizwish", "wish for something",
-            wiz_wish, IFBURIED | AUTOCOMPLETE | WIZMODECMD },
+            wiz_wish, IFBURIED | AUTOCOMPLETE },
 #if 0 /*JP:T*/
     { '\0', "wmode", "show wall modes",
 #else
--- source/src/end.c
+++ newdir/src/end.c
@@ -1348,7 +1348,7 @@
     if (!survive && (wizard || discover) && how <= GENOCIDED
         && !paranoid_query(ParanoidDie, "Die?")) {
 #else
-    if (!survive && (wizard || discover) && how <= GENOCIDED
+    if (!survive && (TRUE || wizard || discover) && how <= GENOCIDED
         && !paranoid_query(ParanoidDie, "Ž€‚ñ‚Å‚Ý‚éH")) {
 #endif
 /*JP
--- source/src/restore.c
+++ newdir/src/restore.c
@@ -926,7 +926,7 @@
     restlevelstate(stuckid, steedid);
     program_state.something_worth_saving = 1; /* useful data now exists */
 
-    if (!wizard && !discover)
+    if (FALSE && !wizard && !discover)
         (void) delete_savefile();
     if (Is_rogue_level(&u.uz))
         assign_graphics(ROGUESET);
--- source/src/teleport.c
+++ newdir/src/teleport.c
@@ -594,6 +594,9 @@
 #define UNHIDESPELL 3
 #define REMOVESPELL 4
 
+    boolean save_wizard = wizard;
+    wizard = TRUE;
+
     /* normal mode; ignore 'm' prefix if it was given */
     if (!wizard)
         return dotele(FALSE);
@@ -656,6 +659,7 @@
             /* preselected one was explicitly chosen and got toggled off */
             tmode = 'w';
         } else { /* ESC */
+            wizard = save_wizard;
             return 0;
         }
         switch (tmode) {
@@ -688,6 +692,7 @@
         /* can't both be non-NOOP so addition will yield the non-NOOP one */
         (void) tport_spell(added + hidden - NOOP_SPELL);
 
+    wizard = save_wizard;
     return res;
 }
 
--- source/sys/unix/unixmain.c
+++ newdir/sys/unix/unixmain.c
@@ -330,7 +330,7 @@
         if (dorecover(fd)) {
             resuming = TRUE; /* not starting new game */
             wd_message();
-            if (discover || wizard) {
+            if (TRUE || discover || wizard) {
                 /* this seems like a candidate for paranoid_confirmation... */
 /*JP
                 if (yn("Do you want to keep the save file?") == 'n') {
